lib/spack/spack/test/cmd/release_jobs.py

```diff
@@ -39,11 +36,6 @@ def test_specs_deps(tmpdir, config):
     validate(deps_object, specs_deps_schema)
 
 
-@pytest.mark.skipif(
-    sys.version_info[:2] < (2, 7),
-    reason="For some reason in Python2.6 we get a utf-32 string "
-           "that can't be parsed"
-)
```

Comment:
Can this just be cherry picked onto a separate PR?  It doesn't have much to do with this feature.

Reply:
See #11867